### ISSUE TITLE
Amazon Seller Partner `GET_LEDGER_SUMMARY_VIEW_DATA` report wrong date format.

### ISSUE THREAD

**Issue by nis130** [airbytehq/airbyte#34830]
### Connector Name

source-amazon-seller-partner

### Connector Version

3.2.1

### What step the error happened?

During the sync

### Relevant information

Error while normalizing `GET_LEDGER_SUMMARY_VIEW_DATA` report since `Date` field in the report is of `MM/DD/YYY` format.

<img width="1101" alt="image" src="https://github.com/airbytehq/airbyte/assets/35602510/30a43b8f-667d-47b6-b077-428d24194451">

### Relevant log output

```shell
of 1 ERROR creating incremental model public.get_ledger_summary_view_data............................................. [ERROR in 0.20s]
Database Error in model get_ledger_summary_view_data (models/generated/airbyte_incremental/public/get_ledger_summary_view_data.sql)
  invalid input syntax for type timestamp with time zone: "04/2023"
1 of 1 ERROR creating incremental model public.get_ledger_summary_view_data............................................. [ERROR in 0.20s]
Database Error in model get_ledger_summary_view_data (models/generated/airbyte_incremental/public/get_ledger_summary_view_data.sql)
  invalid input syntax for type timestamp with time zone: "04/2023",retryable=<null>,timestamp=1707111250288,additionalProperties={}], io.airbyte.config.FailureReason@50734cea[failureOrigin=normalization,failureType=system_error,internalMessage=invalid input syntax for type timestamp with time zone: "04/2023",externalMessage=Normalization failed during the dbt run. This may indicate a problem with the data itself.,metadata=io.airbyte.config.Metadata@4e6280de[additionalProperties={attemptNumber=0, jobId=581, from_trace_message=true}],stacktrace=AirbyteDbtError: 
1 of 1 ERROR creating incremental model public.get_ledger_summary_view_data............................................. [ERROR in 0.20s]
Database Error in model get_ledger_summary_view_data (models/generated/airbyte_incremental/public/get_ledger_summary_view_data.sql)
  invalid input syntax for type timestamp with time zone: "04/2023"
1 of 1 ERROR creating incremental model public.get_ledger_summary_view_data............................................. [ERROR in 0.20s]
Database Error in model get_ledger_summary_view_data (models/generated/airbyte_incremental/public/get_ledger_summary_view_data.sql)
  invalid input syntax for type timestamp with time zone: "04/2023",retryable=<null>,timestamp=1707111250288,additionalProperties={}]],additionalProperties={}]
```

### Contribute

- [ ] Yes, I want to contribute

**Comment by askarpets** [comment]
Hello @nis130,
Would you be able to share your config please so we could replicate and fix this issue?

Thanks!

**Comment by nis130** [comment]
Hey @askarpets,

There is not much of a custom config, I enabled the `GET_LEDGER_SUMMARY_VIEW_DATA` stream on a fresh connection without any `reportOptions`. Since there are no report options amazon uses the default `aggregatedByTimePeriod` as `Montly`. This results in the `Date` field containing just `MM/YYYY`.

**Comment by askarpets** [comment]
@nis130, this issue should be fixed in version `3.2.2`. Could you please upgrade the connector and check if it helps?

**Comment by askarpets** [comment]
Closing the ticket as no response from the customer. Please reopen if needed.
